site/content/pages/community.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Community
+layout: community
+url: /community
+
+headline: Be a part of building the CMS of the future.
+subhead: We're serious about being community driven, so everyone is welcome to join our open, bi-weekly planning sessions.
+primarycta: "[Register on Eventbrite →](https://www.eventbrite.com/e/netlify-cms-planning-session-bi-weekly-tickets-35794058994)"
+
+upcomingevent:
+  hook: The next development planning session is on
+  date: 2017-07-19
+  time: 8am PT
+
+howitworks: Every other week we have public development planning sessions. They're web based, last about an hour, and are geared toward contributors and those interested in contributing. Sessions currently take place every other Wednesday, 8am - 9am PT.
```

Comment:
We should probably add a "how to register" section between how it works and how to join, just to keep that front and center.

Reply:
@erquhart @verythorough what would the "how to register" copy contain? hoping to merge this today if possible.

Reply:
Could use the copy from the [wiki page](https://github.com/netlify/netlify-cms/wiki), but I'm wondering if it's necessary with two "Register now" links just above it on the page. What do you think @erquhart?

I would, however, suggest adding a blurb for the meeting notes:
> **Notes & Agendas**
Check out notes from previous meetings and the agenda for the next one on the [Netlify CMS repo wiki](https://github.com/netlify/netlify-cms/wiki).